### PR TITLE
Basic grid system

### PR DESCRIPTION
**Pull request by GusBaamonde**
- **Creation of grid system**
- **Creation of columns for each device and its breakpoints**

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

dp-grid-system

**Functioning:**
grid-doppler

### CODE REVIEW

File: assets/scss/modules/_grid-system.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    max-width: 1260px;
+    background: $dp-color-snow;
+  }
+}
+
+
+
+
+
+
```
Comment (cbernat) [inline assets/scss/modules/_grid-system.scss]: Some extra space here, maybe you can remove it.

File: assets/scss/templates/_guidelines.scss [hunk 2]
```diff
@@ -235,6 +235,7 @@ body {
     height: 100%;
     transform-origin: top;
     animation: introOpen 0.5s ease-in-out forwards;
+    //animation: ease-in-out forwards;
```
Comment (mvillaseco) [inline assets/scss/templates/_guidelines.scss]: commented line

File: assets/scss/modules/_grid-system.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+@for $i from 1 through 12 {
+  .col-#{$i} {
+    max-width: columns($i, 12);
+    flex: 0 0 columns($i, 12);
+  }
+}
+
+@media screen and (min-width: 320px) {
+  @for $i from 1 through 12 {
+    .col-sm-#{$i} {
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_grid-system.scss]: Consulta para entender, no logro entender la diferencia de los mediaqueries de las clases. No termina generando el mismo max-width y flex que el `col-` comun?

File: assets/scss/modules/_grid-system.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  }
+  .dp-container {
+    max-width: 1260px;
+    background: $dp-color-snow;
+  }
+}
+
+.dp-container-fluid {
+  width: 100%;
+  padding-right: 12px;
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_grid-system.scss]: para los espacios no manejabamos variables?

File: assets/scss/modules/_grid-system.scss [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+[class*='col-'] {
+  position: relative;
+  width: 100%;
+  padding-right: 12px;
+  padding-left: 12px;
+}
+
+.dp-box-shadow {
+  border-radius: 3px;
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_grid-system.scss]: podría ir en una variable ya que se usa en muchos lados

File: assets/scss/modules/_grid-system.scss [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  width: 100%;
+  max-width: 100%;
+  min-height: 100%;
+  border: 1px solid #ccc;
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_grid-system.scss]: el color del borde debería usarse el de las variables

File: assets/scss/modules/_grid-system.scss (hunk 4 above)
Comment (GusBaamonde) [reply assets/scss/modules/_grid-system.scss]: Me los pase, ya los agrego!

File: assets/scss/modules/_grid-system.scss [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+@media screen and (min-width: 320px) {
+  @for $i from 1 through 12 {
+    .col-sm-#{$i} {
+      max-width: columns($i, 12);
+      flex: 0 0 columns($i, 12);
+    }
+  }
+  .dp-container {
+    max-width: 100%;
+    background: $dp-color-lemonyellow;
```
Comment (andresmoschini) [inline assets/scss/modules/_grid-system.scss]: What Is the intent of this class `dp-container`? is it fine having a background color associated?

Comment (GusBaamonde) [reply assets/scss/modules/_grid-system.scss]: We need a container to handle the widths of the columns and create them. The width of the container is handled with the @media queries.
The background color is only to display the @media queries

**Pull request merged**